db/migrate/20151026205814_create_list_entries.rb

```diff
@@ -0,0 +1,10 @@
+class CreateListEntries < ActiveRecord::Migration
+  def change
+    create_table :list_entries do |t|
+      t.references :list, index: true, foreign_key: true, null: false
```

Comment:
I guess this should have a unique compound index to prevent the same book being added twice to the same list.